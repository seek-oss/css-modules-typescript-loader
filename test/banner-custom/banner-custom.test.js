const fs = require('fs');
const compiler = require('../compiler.js');

test('Can add custom banners', async () => {
  await compiler(require.resolve('./index.js'), {
    banner: "// @generated",
  });

  const declaration = fs.readFileSync(
    require.resolve('./index.css.d.ts'),
    'utf-8'
  );

  expect(declaration).toMatchSnapshot();
});
