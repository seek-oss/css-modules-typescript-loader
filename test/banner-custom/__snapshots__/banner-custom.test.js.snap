// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can add custom banners 1`] = `
"// @generated
interface CssExports {
  'foo': string;
}
export const cssExports: CssExports;
export default cssExports;
"
`;
